dipy/reconst/ivim.py

```diff
@@ -226,13 +252,17 @@ def __init__(self, gtab, split_b_D=400.0, split_b_S0=200., bounds=None,
             raise ValueError(b0_s)
 
         ReconstModel.__init__(self, gtab)
-        self.split_b_D = split_b_D
-        self.split_b_S0 = split_b_S0
-        self.bounds = bounds
-        self.two_stage = two_stage
-        self.tol = tol
-        self.options = options
-        self.x_scale = x_scale
+        self.args = args
```

Comment:
What is the goal of `self.args` ? It is never used

Reply:
Will take it down :+1: Thank you for the catch!